sample-api-calls/Data Quality/DistructErrorStaging.sql

```diff
@@ -0,0 +1,25 @@
+
+USE EdFi_Ods_2022
+GO
+
```

Comment:
are we ok with hardcoding the database names? we would have to remember to update if we change the name/year

Reply:
I removed it . Thanks